### PR TITLE
remove simple, easily and other words from documentation

### PR DESCRIPTION
**Pull request by atomfrede**
This removes mostly things 

* Of course
* easily
* just 
* simply

### CODE REVIEW

File: pages/api-gateway.md [hunk 1]
```diff
@@ -75,16 +75,16 @@ Unlike JWT, those tokens are not self-sufficient, and should be stateful, which
...
   - If you want more information on this "user info" request, it is configured using the standard Spring Boot configuration key `security.oauth2.resource.userInfoUri` in your `src/main/resources/application.yml` configuration file.
 - Authentication is not automatically synchronized between the application and Keycloak. Please note that this the standard OpenID Connect workflow, and that we expect to do some specific improvements in JHipster on this matter. As a result:
-  - When a user logs out of the application, he will be automatically logged in again if he refreshes his browser: this is because he is still logged in Keycloak, which provides automatic authentication.
-  - When a user's session is invalidated in Keycloak, if the user is already logged into the application, he will still be able to use the application for a while. This is because OpenID Connect is a stateful mechanism, and the application doesn't know immediately that the session has been invalidated.
+  - When users log out of the application, they will be automatically logged in again if they refreshes the browser: this is because the user is still logged in Keycloak, which provides automatic authentication.
```
Comment (clement26695) [inline pages/api-gateway.md]: I think it is "they refresh" and not "they refreshes"

File: pages/api-gateway.md [hunk 2]
```diff
@@ -75,16 +75,16 @@ Unlike JWT, those tokens are not self-sufficient, and should be stateful, which
...
+  - When a user's session is invalidated in Keycloak, if the user is already logged into the application, they will still be able to use the application for a while. This is because OpenID Connect is a stateful mechanism, and the application doesn't know immediately that the session has been invalidated.
 
 ### JHipster UAA
 
 JHipster provides the option to generate a "UAA" (User Account and Authentication) server, based on Spring Security. This server provides OAuth2 tokens for securing the gateway.
 
 You will find all UAA-related information on our specific [JHipster UAA documentation]({{ site.url }}/using-uaa/).
 
-Then, the gateway uses Spring Security's JWT implementation to send JWT tokens to the microservices, so this works basically the same as with the JWT configuration detailed above.
+Then, the gateway uses Spring Security's JWT implementation to send JWT tokens to the microservices, so this works similar as with the JWT configuration detailed above.
```
Comment (clement26695) [inline pages/api-gateway.md]: similarly
```suggestion
Then, the gateway uses Spring Security's JWT implementation to send JWT tokens to the microservices, so this works similarly as with the JWT configuration detailed above.
```

File: pages/jhipster_uml.md [hunk 3]
```diff
@@ -112,7 +112,7 @@ JHipster UML is a tool that _can_ be used with the JHipster's generator. If you'
 ***
 
 # <a name="howtouse"></a>How to use it
-JHipster-UML is quite easy to use, you only need a class diagram exported in XMI and JHipster-UML will parse it to create your entities.
+JHipster-UML is not hard to use, you only need a class diagram exported in XMI and JHipster-UML will parse it to create your entities.
```
Comment (clement26695) [inline pages/jhipster_uml.md]: Is there another way to say it without using hard ? Using intuitive for instance ? Or maybe just remove the first part of the sentence ?

File: pages/jhipster_uml.md [hunk 4]
```diff
@@ -500,28 +500,28 @@ Here is the [repo](https://github.com/guillaumefinance/MDG-Sparx-EA-UML-JHipster
 JHipster and JHipster UML support both support enumerations.
 Here's how you define them:
   - For Modelio, drag and drop the `Enumeration` object and place it somewhere. Finally, add the `Enumeration Literal` object to the enum to add it;
-  - For UML Designer, there is the `Enumeration` object that can be placed and used. However, the literal is not called `Enumeration Literal` but just `Literal`;
+  - For UML Designer, there is the `Enumeration` object that can be placed and used. However, the literal is not called `Enumeration Literal` but on;y `Literal`;
```
Comment (clement26695) [inline pages/jhipster_uml.md]: typo here
```suggestion
  - For UML Designer, there is the `Enumeration` object that can be placed and used. However, the literal is not called `Enumeration Literal` but only `Literal`;
```

File: pages/jhipster_uml.md [hunk 5]
```diff
@@ -551,26 +551,26 @@ We follow the same guidelines as JHipster, with a few additions:
 
 ## <a name="parsermodifications"></a>Parser modifications
 
-The 1.0.0 release brings a new parser system making any change (parser creation, update, deletion) ultra-easy, provided the XMI is easy to parse.
+The 1.0.0 release brings a new parser system making any change (parser creation, update, deletion) hazzle-free, provided the XMI can be parsed.
```
Comment (clement26695) [inline pages/jhipster_uml.md]: Do you mean:
```suggestion
The 1.0.0 release brings a new parser system making any change (parser creation, update, deletion) hassle-free, provided the XMI can be parsed.
```

File: pages/kubernetes.md [hunk 6]
```diff
@@ -137,7 +137,7 @@ Consequently, for microservices applications, the JHipster Kubernetes sub-genera
 
 ### Managing the JHipster Registry or Consul in Kubernetes
 
-For the JHipster Registry and Consul, [StatefulSets](https://kubernetes.io/docs/concepts/abstractions/controllers/statefulsets/) configurations are provided. Those are a special kind of Kubernetes resources that can handle stateful applications and will let you scale your service registries for high availability. For more information on high-availability for Eureka and Consul refer to their respective documentation.
+For the JHipster Registry and Consul, [StatefulSets](https://kubernetes.io/docs/concepts/abstractions/controllers/statefulsets/) configurations are provided. Those are a certain kinds of Kubernetes resources which can handle stateful applications and will let you scale your service registries for high availability. For more information on high-availability for Eureka and Consul refer to their respective documentation.
```
Comment (clement26695) [inline pages/kubernetes.md]: Is there a 's' here ? I would use
```suggestion
For the JHipster Registry and Consul, [StatefulSets](https://kubernetes.io/docs/concepts/abstractions/controllers/statefulsets/) configurations are provided. Those are a certain kind of Kubernetes resources which can handle stateful applications and will let you scale your service registries for high availability. For more information on high-availability for Eureka and Consul refer to their respective documentation.
```

File: pages/separating-front-end-and-api.md [hunk 7]
```diff
@@ -132,7 +132,7 @@ This configuration will require some tuning depending on your specific needs, bu
 ### Configuration with Oauth 2.0 and traefik
 Here is a sample `site.conf` for Oauth 2.0:
 if the server base name is `back`, and the name of the server hosting traefik is `api.jhipster.tech`.
-If this configuration is for a docker image, don't use `localhost` instead of `api.jhipster.tech` as it's resolv in container, not your host.
+If this configuration is for a docker image, don't use `localhost` instead of `api.jhipster.tech` as it is resolved in a container, not your host.
```
Comment (clement26695) [inline pages/separating-front-end-and-api.md]: if we remove host everywhere, we should maybe remove it from here too

File: pages/jhipster_uml.md (hunk 3 above)
Comment (atomfrede) [reply pages/jhipster_uml.md]: In fact this was one of tricky parts. Will update it. Thanks

**Comment by atomfrede** [comment]
Thanks @clement26695 for the review, will update the remaining points later.

File: pages/api-gateway.md [hunk 8]
```diff
@@ -75,16 +75,16 @@ Unlike JWT, those tokens are not self-sufficient, and should be stateful, which
...
   - If you want more information on this "user info" request, it is configured using the standard Spring Boot configuration key `security.oauth2.resource.userInfoUri` in your `src/main/resources/application.yml` configuration file.
 - Authentication is not automatically synchronized between the application and Keycloak. Please note that this the standard OpenID Connect workflow, and that we expect to do some specific improvements in JHipster on this matter. As a result:
-  - When a user logs out of the application, he will be automatically logged in again if he refreshes his browser: this is because he is still logged in Keycloak, which provides automatic authentication.
-  - When a user's session is invalidated in Keycloak, if the user is already logged into the application, he will still be able to use the application for a while. This is because OpenID Connect is a stateful mechanism, and the application doesn't know immediately that the session has been invalidated.
+  - When users log out of the application, they will be automatically logged in again if they refresh the browser: this is because the user is still logged in Keycloak, which provides automatic authentication.
```
Comment (vishal423) [inline pages/api-gateway.md]: I think this paragraph is not applicable anymore as we do logout from the OpenID provider.

Comment (atomfrede) [reply pages/api-gateway.md]: Let me check. It thats the case we can remove that paragraph

Comment (atomfrede) [reply pages/api-gateway.md]: At least for monolith we log out the user from keycloak.

Comment (atomfrede) [reply pages/api-gateway.md]: Also on a gateway application the user is logged out and refreshing the browser does not result in logged in automatically, so I removed the paragraph.

**Comment by atomfrede** [comment]
I would consider this no ready to be merged.

**Comment by jdubois** [comment]
Awesome work, thank you so much! Many of those issues were my fault.

**Pull request merged**